pkg/provider/azure_loadbalancer.go

```diff
@@ -1653,7 +1658,15 @@ func (az *Cloud) reconcileLoadBalancer(clusterName string, service *v1.Service,
 
 	lbName := *lb.Name
 	lbResourceGroup := az.getLoadBalancerResourceGroup()
+	lbPodBackendPoolIDs := map[bool][]string{}
+
+	if az.LoadBalancerBackendPoolConfigurationType == consts.LoadBalancerBackendPoolConfigurationTypePodIP {
+		lbPodBackendPoolIDs = az.getPodBackendPoolIDsForService(service, lbName)
+
+	}
+
 	lbBackendPoolIDs := az.getBackendPoolIDsForService(service, clusterName, lbName)
```

Comment:
Shouldn't this be under else??

Reply:
These are two different variables so it need not be in else, anyway this is just for testing and we'll change this once we figure out what to do about lb rules